### PR TITLE
Docs - pin python package markdown at 3.2.1

### PR DESCRIPTION
**Pull request by jeremylt**
@jedbrown, can you verify that this fixes the issue on readthedocs? Locally, I have v3.0.1, so I believe that you're correct and this is the problem.

### CODE REVIEW

File: doc/sphinx/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 sphinx>=3.0
 breathe>=4.17
+markdown==3.0.1
```
Comment (valeriabarra) [inline doc/sphinx/requirements.txt:3]: Was `3.2.1` the one that passed?

Comment (jeremylt) [reply doc/sphinx/requirements.txt]: Good catch, thanks. 3.0.1 is what I had locally. 3.2.1 is working locally as well.

**Comment by jedbrown** [comment]
Building here: https://readthedocs.org/projects/libceed/builds/11077011/

**Comment by jeremylt** [comment]
@jedbrown thanks. It looks like readthedocs is happy. I'll merge when CI is finished.

**Pull request merged**